metron-analytics/metron-statistics/src/main/java/org/apache/metron/statistics/StellarStatisticsFunctions.java

```diff
@@ -425,4 +428,74 @@ public Object apply(List<Object> args) {
       return result;
     }
   }
+
+  /**
+   * Calculates the statistical bin that a value falls in.
+   */
+  @Stellar(namespace = "STATS", name = "BIN"
+          , description = "Computes the bin that the value is in based on the statistical distribution."
+          , params = {
+          "stats - The Stellar statistics object"
+          , "value - The value to bin"
+          , "range? - A list of percentile bin ranges (excluding min and max) or a string representing a known and common set of bins.  " +
+          "For convenience, we have provided QUARTILE, QUINTILE, and DECILE which you can pass in as a string arg." +
+          " If this argument is omitted, then we assume a Quartile bin split."
+  }
+          , returns = "Which bin the value falls in such that bin < value < bin + 1"
```

Comment:
One of the "<" has to be "<=", please.  From code, it's the upper one.

Suggest phrasing as:
returns = "Which bin N the value falls in such that bound(N-1) < value <= bound(N). " +
"No min and max bounds are provided, so values smaller than the 0'th bound go in the 0'th bin, " +
"and values greater than the last bound go in the M'th bin."